### PR TITLE
Stop using knative.dev/serving in cmd/ and pkg/

### PR DESCRIPTION
**Pull request by tcnghia**
This should leave conformance testing as the only knative.dev/serving dependency left. I will follow up with another PR.

### CODE REVIEW

File: pkg/reconciler/ingress/resources/util.go [hunk 1]
```diff
@@ -32,6 +32,12 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+const (
+	ServingGroupName       = "serving.knative.dev"
```
Comment (vagababov) [inline pkg/reconciler/ingress/resources/util.go:36]: You have this string declared twice?

File: pkg/defaults/deployment_defaults.go [hunk 2]
```diff
@@ -23,8 +23,23 @@ import (
...
 	"knative.dev/pkg/apis"
+)
+
+const (
+	// ServingGroupName is the group name for knative labels and annotations
+	ServingGroupName = "serving.knative.dev"
+
+	// ServingServiceLabelKey is the label key attached to a Route and Configuration indicating by
+	// which Service they are created.
+	ServingServiceLabelKey = ServingGroupName + "/service"
```
Comment (vagababov) [inline pkg/defaults/deployment_defaults.go:34]: Those don't seem to be used crosspackage, so probably lowecase?

**Comment by vagababov** [comment]
LG, but tests look unhappy

File: pkg/defaults/deployment_defaults.go (hunk 2 above)
Comment (tcnghia) [reply pkg/defaults/deployment_defaults.go]: Done

File: pkg/reconciler/ingress/resources/util.go (hunk 1 above)
Comment (tcnghia) [reply pkg/reconciler/ingress/resources/util.go]: Yeah, the defaults package and the reconciler/ packages are quite independent so I didn't want to refer to one from the other.

**Pull request merged**